cli/api.py

```diff
@@ -8,13 +8,21 @@ def executor(args: 'Namespace'):
 
     :param args: arguments coming from the CLI.
     """
-    from jina.peapods.pods.factory import PodFactory
-
-    try:
-        with PodFactory.build_pod(args) as p:
-            p.join()
-    except KeyboardInterrupt:
-        pass
+    uses = args.uses
+    if (
+        uses.startswith('jinahub+docker://')
+        or uses.startswith('docker://')
+        or uses.startswith('jinahub://')
+    ):
+        from jina.peapods.pods.factory import PodFactory
+
+        try:
+            with PodFactory.build_pod(args) as p:
+                p.join()
+        except KeyboardInterrupt:
```

Comment:
I think this is already handled inside the Pod context manager

Reply:
what is already handled?

Reply:
keyboard interrupt?

Reply:
yes, handled, confirm